src/crimson/osd/backfill_facades.h

```diff
@@ -42,6 +42,10 @@ struct BackfillState::PeeringFacade {
     return peering_state.update_complete_backfill_object_stats(hoid, stats);
   }
 
+  decltype(auto) is_backfilling() const {
```

Comment:
nit, might want to s/decltype(auto)/bool/ for better readability? i understand that `PeeringFacade` is proxying the `is_backfilling()` calls to underlying `peering_state`, and`decltype(auto)` conveys the idea of "forwarding the call" perfectly. but i think the type is also part of the documentation, probably `bool` helps the reader to get the point more quickly?